modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -27,6 +29,8 @@ Show Modal Title Line:: Shows the title line in the access request modal.
 
 The API Card component shows information about an API, including icon, name, and description.
 
+Data shown from Anypoint Platform: API icon, name, and description from xref:exchange::index.adoc[Anypoint Exchange].
+
```

Comment:
Reading this again made me realize that the Icon, name and description come from the API publication config in ACM (done in the ACM control panel when publishing an API). I suggest we remove the "data shown from Anypoint Platform line" and say something like this instead:

"The API icon, name and description shown in this component are the ones defined when the API version was added to the community in ACM control panel" (and link to the section where API publishing/editing is described.

Reply:
Updated.